bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/serial.xml

```diff
@@ -34,6 +38,24 @@
 				<description>Seconds between connect retries when KNX link has been lost, 0 means never retry</description>
 				<default>0</default>
 			</parameter>
+			<parameter name="useCemi" type="boolean">
+				<label>Use CEMI</label>
+				<description>Use newer CEMI frame format. May be useful for newer serial devices like KNX RF sticks, kBerry, etc.</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="keyringFile" type="text" groupName="knxsecure">
```

Comment:
Wouldn't it make sense to add that directly to the JSON database (i.e. add the file content directly here instead of a filename).